src/Platform.Handlers/src/Xamarin.Platform.Handlers/Xamarin.Platform.Handlers.csproj

```diff
@@ -2,12 +2,20 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.1;netstandard2.0;Xamarin.iOS10;MonoAndroid10.0</TargetFrameworks>
     <RootNamespace>Xamarin.Platform</RootNamespace>
-    <AssemblyName>Xamarin.Platform</AssemblyName> 
+    <AssemblyName>Xamarin.Platform</AssemblyName>
     <Nullable>enable</Nullable>
   </PropertyGroup>
   <PropertyGroup>
     <PackageId>Xamarin.Platform</PackageId>
     <Product>Xamarin</Product>
   </PropertyGroup>
   <Import Project="..\..\..\..\.nuspec\Xamarin.Forms.MultiTargeting.targets" />
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Hosting" Version="5.0.0-rc.1.20451.14" />
```

Comment:
The full Hosting package is pretty heavy weight and has a lot of dependencies.

Do you need to reference `Microsoft.Extensions.Hosting` or would it be better if you referenced `Microsoft.Extensions.Hosting.Abstractions` and `Microsoft.Extensions.DependencyInjection.Abstractions`?